### DISCUSSION TITLE
Implement dimming of inactive windows

### DISCUSSION THREAD

**Discussion by alexdev16** [YaLTeR/niri#635] [Ideas]
For inactive windows, it would be great to be able to adjust the color saturation and brightness. The dimming should be smooth. In my opinion, it's nice and useful for those who don't like the focus-ring and border.

**Comment by YaLTeR** [comment]
This requires long-running shaders, which is not something currently implemented. I think once we get long-running shaders, it might make sense to leave these features to custom shaders.

**Comment by cyuria** [comment]
Very similar, but I usually add a bit of transparency to my inactive windows using a custom window rule. This might be a bit of a stop gap fix until the long running shaders or another solution is implemented.
